node_modules/oae-principals/lib/test/util.js

```diff
@@ -270,6 +270,18 @@ var assertGetUserFails = module.exports.assertGetUserFails = function(restCtx, u
 var assertGetMeSucceeds = module.exports.assertGetMeSucceeds = function(restCtx, callback) {
     RestAPI.User.getMe(restCtx, function(err, me, response) {
         assert.ok(!err);
+
+        // Sanity check the me model
+        assert.ok(me);
+        assert.ok(me.tenant);
+        assert.ok(_.isString(me.tenant.alias));
+        assert.ok(_.isString(me.tenant.displayName));
+        if (me.tenant.alias === 'guest') {
```

Comment:
Ideally yes, but we currently don't expose this information anywhere and we hardcode `'admin'` in all other areas where the global admin tenant alias is needed.